packages/tokamak/sdk/package.json

```diff
@@ -63,7 +63,6 @@
   "dependencies": {
     "@eth-optimism/contracts": "0.6.0",
     "@eth-optimism/contracts-bedrock": "workspace:*",
-    "@tokamak-network/titan2-contracts": "workspace:*",
```

Comment:
I think we should better use titan2-contracts although interface is same. How about changing all dependency to use titan2-contracts instead of contracts-bedrock in tokamak/sdk?

Reply:
Thank you!
Let me update.
Actually, i don't have good understanding about js, npm. I am learning in this job.

Reply:
Thank you very much!
I think you get many knowledge in short time. 😁 You are so smart and fashionated! 😍